README.md

```diff
@@ -10,6 +10,7 @@ Ensure that the following prerequisites are met before using the SDK:
 
 - [Python](https://www.python.org) 3.7 or newer must be installed on your machine.
```

Comment:
Maybe for some time until api v1.2 is entirely deprecated, it might be good to add a NOTE before prereq section that inform users that this only works with API v1.3, and for users who need to use api v1.2, rather use sdk version X